public/app/features/canvas/runtime/scene.tsx

```diff
@@ -56,9 +68,17 @@ export class Scene {
   currentLayer?: FrameState;
   isEditingEnabled?: boolean;
   skipNextSelectionBroadcast = false;
+  contextMenu: ContextMenuState;
 
   constructor(cfg: CanvasFrameOptions, enableEditing: boolean, public onSave: (cfg: CanvasFrameOptions) => void) {
     this.root = this.load(cfg, enableEditing);
+    this.contextMenu = {
+      isMenuVisible: false,
+      anchorPoint: {
+        x: 0,
+        y: 0,
+      },
+    };
```

Comment:
This is unrelated to these change but I noticed that when opening up the inline editor on a 14 inch MBP the inline editor is not accessible (able to be dragged around) and as a result not really usable

Probably can make this a separate glitch / bug in https://github.com/grafana/grafana/issues/47740 epic

https://user-images.githubusercontent.com/22381771/168715376-9e343ce3-7528-4518-bc1a-3e8047ce9718.mov

Reply:
I'll make a bug for it. Not sure, but it might be related to the issue I noticed with our elements - when moving to a smaller screen, some of them cannot be seen.